doc/release/RELEASE-NOTES.md

```diff
@@ -19,6 +23,10 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 ### What's new
 
 ### Release instructions:
+* Recreate index with new schema:
+    1. Before deploying to the env, especially for Production and Staging, you must first delete the opensearch production location index through EС2 bastion.
+    2. Delete the EFS storage from the ran file or the file through the EС2 bastion instance.
+    3. Start deploying changes to the env.
```

Comment:
Hi @Innavin369,
1. It will be possible to perform these instructions only after merging this PR (https://github.com/opensupplyhub/open-supply-hub/pull/322), as it enables access from the EC2 bastion to EFS and OpenSearch.
2. Could you please ensure that there is no need to stop Logstash before running the deployment to AWS? We need to be certain that the running Logstash app will reindex OpenSearch from scratch if the EFS is cleared and the production locations index is deleted in OpenSearch. You can test this locally, and then please leave the research result under this comment.

Reply:
1. That PR has already been merged.
2. I tested the latest update locally without stopping a running Logstash app and it did not reindex OpenSearch. logstash_jdbc_last_run has been created but a new index hasn't been created. I conclude that we need to stop the Logstash app.

Reply:
> I conclude that we need to stop the Logstash app.

Okay, please add to the release instructions that Logstash should be stopped before deployment.